diff --git a/node_modules/@vendure/dashboard/index.html b/node_modules/@vendure/dashboard/index.html
index 46107c4..212372d 100644
--- a/node_modules/@vendure/dashboard/index.html
+++ b/node_modules/@vendure/dashboard/index.html
@@ -2,6 +2,7 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8" />
+    <title>Khukuri House</title>
     <link rel="icon" type="image/png" href="/favicon.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="description" content="Vendure Admin Dashboard" />
diff --git a/node_modules/@vendure/dashboard/src/app/common/use-page-title.ts b/node_modules/@vendure/dashboard/src/app/common/use-page-title.ts
index 2a1354f..02abd62 100644
--- a/node_modules/@vendure/dashboard/src/app/common/use-page-title.ts
+++ b/node_modules/@vendure/dashboard/src/app/common/use-page-title.ts
@@ -1,7 +1,6 @@
-import { useMatches } from '@tanstack/react-router';
-import React, { isValidElement, ReactElement, useEffect, useState } from 'react';
+import React, { isValidElement, ReactElement } from 'react';
 
-const DEFAULT_TITLE = 'Vendure';
+const DEFAULT_TITLE = 'Khukuri House';
 
 /**
  * @description
@@ -9,18 +8,7 @@ const DEFAULT_TITLE = 'Vendure';
  * data from the route loader.
  */
 export function usePageTitle() {
-    const matches = useMatches();
-    const [pageTitle, setPageTitle] = useState<string>(DEFAULT_TITLE);
-
-    useEffect(() => {
-        const lastMatch = matches.at(-1);
-        const breadcrumb = (lastMatch?.loaderData as any)?.breadcrumb;
-
-        const breadcrumbTitle = normalizeBreadcrumb(breadcrumb);
-        setPageTitle([breadcrumbTitle, DEFAULT_TITLE].filter(x => !!x).join(' • '));
-    }, [matches]);
-
-    return pageTitle;
+    return DEFAULT_TITLE;
 }
 
 const renderNodeAsString = function (reactNode: React.ReactNode): string {
diff --git a/node_modules/@vendure/dashboard/src/app/routeTree.gen.ts b/node_modules/@vendure/dashboard/src/app/routeTree.gen.ts
index 275c926..5ad0dfe 100644
--- a/node_modules/@vendure/dashboard/src/app/routeTree.gen.ts
+++ b/node_modules/@vendure/dashboard/src/app/routeTree.gen.ts
@@ -395,8 +395,8 @@ const AuthenticatedProductsProductsProductIdOptionGroupsProductOptionGroupIdOpti
   )
 
 export interface FileRoutesByFullPath {
-  '/login': typeof LoginRoute
   '/': typeof AuthenticatedIndexRoute
+  '/login': typeof LoginRoute
   '/administrators': typeof AuthenticatedAdministratorsAdministratorsRoute
   '/assets': typeof AuthenticatedAssetsAssetsRoute
   '/channels': typeof AuthenticatedChannelsChannelsRoute
@@ -567,8 +567,8 @@ export interface FileRoutesById {
 export interface FileRouteTypes {
   fileRoutesByFullPath: FileRoutesByFullPath
   fullPaths:
-    | '/login'
     | '/'
+    | '/login'
     | '/administrators'
     | '/assets'
     | '/channels'
@@ -753,7 +753,7 @@ declare module '@tanstack/react-router' {
     '/_authenticated': {
       id: '/_authenticated'
       path: ''
-      fullPath: ''
+      fullPath: '/'
       preLoaderRoute: typeof AuthenticatedRouteImport
       parentRoute: typeof rootRouteImport
     }
diff --git a/node_modules/@vendure/dashboard/src/app/routes/__root.tsx b/node_modules/@vendure/dashboard/src/app/routes/__root.tsx
index 30d4bff..67ce514 100644
--- a/node_modules/@vendure/dashboard/src/app/routes/__root.tsx
+++ b/node_modules/@vendure/dashboard/src/app/routes/__root.tsx
@@ -13,7 +13,7 @@ export const Route = createRootRouteWithContext<MyRouterContext>()({
 });
 
 function RootComponent() {
-    document.title = usePageTitle();
+    document.title = usePageTitle() || 'Khukuri House';
     return (
         <>
             <HeadContent />
diff --git a/node_modules/@vendure/dashboard/src/lib/components/layout/app-sidebar.tsx b/node_modules/@vendure/dashboard/src/lib/components/layout/app-sidebar.tsx
index 566ccff..abced7e 100644
--- a/node_modules/@vendure/dashboard/src/lib/components/layout/app-sidebar.tsx
+++ b/node_modules/@vendure/dashboard/src/lib/components/layout/app-sidebar.tsx
@@ -12,6 +12,7 @@ import { getNavMenuConfig } from '@/vdb/framework/nav-menu/nav-menu-extensions.j
 import { useDisplayLocale } from '@/vdb/hooks/use-display-locale.js';
 import * as React from 'react';
 import { ChannelSwitcher } from './channel-switcher.js';
+import { PermissionGuard } from '@/vdb/components/shared/permission-guard.js';
 
 export function AppSidebar({ ...props }: React.ComponentProps<typeof Sidebar>) {
     const { extensionsLoaded } = useDashboardExtensions();
@@ -22,7 +23,9 @@ export function AppSidebar({ ...props }: React.ComponentProps<typeof Sidebar>) {
         extensionsLoaded && (
             <Sidebar collapsible="icon" {...props} side={isRTL ? 'right' : 'left'}>
                 <SidebarHeader>
-                    <ChannelSwitcher />
+                    <PermissionGuard requires={['Superadmin']}>
+                        <ChannelSwitcher />
+                    </PermissionGuard>
                 </SidebarHeader>
                 <SidebarContent className="flex flex-col h-full overflow-y-auto">
                     <NavMain items={sections} />
diff --git a/node_modules/@vendure/dashboard/src/lib/components/layout/nav-user.tsx b/node_modules/@vendure/dashboard/src/lib/components/layout/nav-user.tsx
index 5d64e6a..117d97b 100644
--- a/node_modules/@vendure/dashboard/src/lib/components/layout/nav-user.tsx
+++ b/node_modules/@vendure/dashboard/src/lib/components/layout/nav-user.tsx
@@ -25,6 +25,7 @@ import { Trans } from '@lingui/react/macro';
 import { useMemo } from 'react';
 import { Dialog, DialogTrigger } from '../ui/dialog.js';
 import { LanguageDialog } from './language-dialog.js';
+import { PermissionGuard } from '@/vdb/components/shared/permission-guard.js';
 
 export function NavUser() {
     const { isMobile, state } = useSidebar();
@@ -92,15 +93,17 @@ export function NavUser() {
                                     </div>
                                 </div>
                             </DropdownMenuLabel>
-                            <DropdownMenuSeparator />
-                            <DropdownMenuGroup>
-                                <DropdownMenuItem asChild>
-                                    <a href="https://vendure.io/pricing" target="_blank">
-                                        <Sparkles />
-                                        <Trans>Explore Enterprise Edition</Trans>
-                                    </a>
-                                </DropdownMenuItem>
-                            </DropdownMenuGroup>
+                            <PermissionGuard requires={['Superadmin']}>
+                                <DropdownMenuSeparator />
+                                <DropdownMenuGroup>
+                                    <DropdownMenuItem asChild>
+                                        <a href="https://vendure.io/pricing" target="_blank">
+                                            <Sparkles />
+                                            <Trans>Explore Enterprise Edition</Trans>
+                                        </a>
+                                    </DropdownMenuItem>
+                                </DropdownMenuGroup>
+                            </PermissionGuard>
                             <DropdownMenuSeparator />
                             <DropdownMenuGroup>
                                 <DropdownMenuItem asChild>
@@ -108,14 +111,16 @@ export function NavUser() {
                                         <Trans>Profile</Trans>
                                     </Link>
                                 </DropdownMenuItem>
-                                <DialogTrigger asChild>
-                                    <DropdownMenuItem className="flex gap-2">
-                                        <div>
-                                            <Trans>Language</Trans>
-                                        </div>
-                                        <div className="text-muted-foreground">{humanReadableLanguage}</div>
-                                    </DropdownMenuItem>
-                                </DialogTrigger>
+                                <PermissionGuard requires={['Superadmin']}>
+                                    <DialogTrigger asChild>
+                                        <DropdownMenuItem className="flex gap-2">
+                                            <div>
+                                                <Trans>Language</Trans>
+                                            </div>
+                                            <div className="text-muted-foreground">{humanReadableLanguage}</div>
+                                        </DropdownMenuItem>
+                                    </DialogTrigger>
+                                </PermissionGuard>
                                 <DropdownMenuSub>
                                     <DropdownMenuSubTrigger>
                                         <Trans>Theme</Trans>
@@ -144,26 +149,28 @@ export function NavUser() {
                                 </DropdownMenuSub>
                             </DropdownMenuGroup>
                             {isDevMode && (
-                                <DropdownMenuSub>
-                                    <DropdownMenuSubTrigger>
-                                        <Trans>Dev Mode</Trans>
-                                    </DropdownMenuSubTrigger>
-                                    <DropdownMenuPortal>
-                                        <DropdownMenuSubContent>
-                                            <DropdownMenuRadioGroup
-                                                value={settings.devMode.toString()}
-                                                onValueChange={value => setDevMode(value === 'true')}
-                                            >
-                                                <DropdownMenuRadioItem value="true">
-                                                    <Trans>Enabled</Trans>
-                                                </DropdownMenuRadioItem>
-                                                <DropdownMenuRadioItem value="false">
-                                                    <Trans>Disabled</Trans>
-                                                </DropdownMenuRadioItem>
-                                            </DropdownMenuRadioGroup>
-                                        </DropdownMenuSubContent>
-                                    </DropdownMenuPortal>
-                                </DropdownMenuSub>
+                                <PermissionGuard requires={['Superadmin']}>
+                                    <DropdownMenuSub>
+                                        <DropdownMenuSubTrigger>
+                                            <Trans>Dev Mode</Trans>
+                                        </DropdownMenuSubTrigger>
+                                        <DropdownMenuPortal>
+                                            <DropdownMenuSubContent>
+                                                <DropdownMenuRadioGroup
+                                                    value={settings.devMode.toString()}
+                                                    onValueChange={value => setDevMode(value === 'true')}
+                                                >
+                                                    <DropdownMenuRadioItem value="true">
+                                                        <Trans>Enabled</Trans>
+                                                    </DropdownMenuRadioItem>
+                                                    <DropdownMenuRadioItem value="false">
+                                                        <Trans>Disabled</Trans>
+                                                    </DropdownMenuRadioItem>
+                                                </DropdownMenuRadioGroup>
+                                            </DropdownMenuSubContent>
+                                        </DropdownMenuPortal>
+                                    </DropdownMenuSub>
+                                </PermissionGuard>
                             )}
                             <DropdownMenuSeparator />
                             <DropdownMenuItem onClick={handleLogout}>
diff --git a/node_modules/@vendure/dashboard/src/lib/framework/defaults.ts b/node_modules/@vendure/dashboard/src/lib/framework/defaults.ts
index 4d46c29..8ba62e7 100644
--- a/node_modules/@vendure/dashboard/src/lib/framework/defaults.ts
+++ b/node_modules/@vendure/dashboard/src/lib/framework/defaults.ts
@@ -13,7 +13,7 @@ export function registerDefaults() {
         sections: [
             {
                 id: 'insights',
-                title: /* i18n*/ 'Insights',
+                title: /* i18n*/ 'Dashboard',
                 placement: 'top',
                 icon: ChartLine,
                 url: '/',
@@ -21,7 +21,7 @@ export function registerDefaults() {
             },
             {
                 id: 'catalog',
-                title: /* i18n*/ 'Catalog',
+                title: /* i18n*/ 'Product',
                 icon: Tags,
                 placement: 'top',
                 order: 200,
@@ -42,7 +42,7 @@ export function registerDefaults() {
                     },
                     {
                         id: 'facets',
-                        title: /* i18n*/ 'Facets',
+                        title: /* i18n*/ 'Category',
                         url: '/facets',
                         order: 300,
                         requiresPermission: ['ReadProduct', 'ReadCatalog'],
